tasks.py

```diff
@@ -92,31 +76,101 @@ def generate_payload(submission_obj, payload_template):
     # TODO: implement this # pylint: disable=fixme
     return {"foo": "bar"}
 
+@celery_app.task(name="tasks.send-csv", bind=True)
+def send_csv(self, external_code, external_system, submission_obj):
+    # pylint: disable=unused-argument
+    """
+        does the work to create a csv and email it
+        records csv in database
```

Comment:
this might just be outdated doc string. looks like the csv is saved into file system versus database?

Reply:
i think i'll leave it in for now.
we should probably save it to the db so that we have a record of it.  the heroku file system is ephemeral and so the files could easily get wiped out and we won't have that history to fall back on.

i'll implement it when i revisit this code to call the email microservice.